src/hotspot/share/prims/jvmtiThreadState.hpp

```diff
@@ -98,6 +98,9 @@ class JvmtiVTMSTransitionDisabler {
   static bool VTMS_notify_jvmti_events()             { return _VTMS_notify_jvmti_events; }
   static void set_VTMS_notify_jvmti_events(bool val) { _VTMS_notify_jvmti_events = val; }
 
+  static int  VTMS_transition_count()                { return _VTMS_transition_count; }
+  static void set_VTMS_transition_count(bool val)    { _VTMS_transition_count = val; }
```

Comment:
Although there is a call to `set_VTMS_transition_count()`, I don't see any calls to `VTMS_transition_count()`. Are these really needed?

Reply:
Thanks.
It was used in an assert that I recently removed as not important. Will remove it as well.